.travis.yml

```diff
@@ -1,5 +1,16 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.0.0
+- 2.0.0
```

Comment:
this is the ruby version. It's not that they had to upgrade; it's what they were using.